cogs/PlaytesterPing.py

```diff
@@ -0,0 +1,98 @@
+import discord
+from discord.ext import commands
+import re
+import requests
+import os
+
+url = "https://api.github.com/graphql"
+githubAccessToken = os.getenv("githubAccessToken")
+
+def getLatestDiscussion():
+    headers = {
+        "Authorization": f"Bearer {githubAccessToken}"
+    }
+
+    query = """
+    query {
+        repository(owner: "R2Northstar", name: "Northstar") {
+            discussions(first: 1) {
+                edges { 
+                    node {
+                        author {
+                            login
+                            url
+                        }
+                        title
+                        number
+                        body
+                        url
+                    }
+                }
+            }
+        }
+    }
+    """
+    try:
+        response = requests.post(url, json={"query": query}, headers=headers)
+        if response.status_code == 200:
+            raw_data = response.json()
+            
+    except requests.exceptions.RequestException as err:
+        print(f"GitHub API request failed: {err}")
+        return None
+    
+    discussion_post = {
+        'author': raw_data["data"]["repository"]["discussions"]["edges"][0]["node"]["author"]["login"],
+        'title': raw_data["data"]["repository"]["discussions"]["edges"][0]["node"]["title"],
+        'body': raw_data["data"]["repository"]["discussions"]["edges"][0]["node"]["body"],
+        'number': raw_data["data"]["repository"]["discussions"]["edges"][0]["node"]["number"],
+        'url': raw_data["data"]["repository"]["discussions"]["edges"][0]["node"]["url"]
+    }
+    
+    return discussion_post
+
+def getLatestReleaseName():
+    headers = {
+        "Authorization": f"Bearer {githubAccessToken}"
+    }
+
+    try:
+        response = requests.get("https://api.github.com/repos/R2Northstar/NorthstarLauncher/releases", headers=headers)
+        if response.status_code == 200:
+            reponse = response
+            
+    except requests.exceptions.RequestException as err:
+        print(f"GitHub API request failed: {err}")
+        return None
+    
+    return response.json()[0]["tag_name"]
+
+class PlayTesterPing(commands.Cog):
+    def __init__(self, bot :commands.Bot) -> None:
+        self.bot = bot
+        
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        if message.author == self.bot.user:
+            return
+
+        if message.channel.id == 939573786355859498: #mod-releases-thunderstore
```

Comment:
Not sure how the rest of Spectre code looks but it would probably be best to define channel ids as a const/global at the top of the file so that it's easier to move them to a config file in the long term ^^

Reply:
other files dont seem to do that, i just made the thunderstore release channel a variable and moved it along with the playtester ping channel variable up a bit